libafl_nyx/src/test.rs

```diff
@@ -0,0 +1,47 @@
+#[allow(unused_imports)]
```

Comment:
Why allow unused imports? Fine for dev but maybe we want to disable this later (?)

Reply:
It's for clippy in macos.

Reply:
Why?

Reply:
you can use `[#cfg(target_os = "linux")]` for imports